crates/router/src/connector/airwallex/transformers.rs

```diff
@@ -0,0 +1,377 @@
+use masking::PeekInterface;
+use serde::{Deserialize, Serialize};
+use time::PrimitiveDateTime;
+use url::Url;
+use uuid::Uuid;
+
+use crate::{
+    core::errors,
+    pii::{self, Secret},
+    services,
+    types::{self, api, storage::enums},
+};
+
+#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+pub struct AirwallexIntentRequest {
+    request_id: String,
+    amount: i64,
+    currency: enums::Currency,
+    merchant_order_id: String,
+}
+impl TryFrom<&types::PaymentsAuthorizeSessionTokenRouterData> for AirwallexIntentRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: &types::PaymentsAuthorizeSessionTokenRouterData,
+    ) -> Result<Self, Self::Error> {
+        Ok(Self {
+            request_id: Uuid::new_v4().to_string(),
+            amount: item.request.amount,
+            currency: item.request.currency,
+            merchant_order_id: Uuid::new_v4().to_string(),
```

Comment:
What this `merchant_order_id` refers to?

Reply:
It is ID created in merchant's order system that corresponds to this PaymentIntent.
Already changed to `payment_id`